### PR TITLE
docs: Add Bento Deployment details docs

### PR DESCRIPTION
**Pull request by Sherlock113**
## What does this PR address?

Add some explanations for the properties on the Deployment details page.

## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, `pre-commit run -a` script has passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [x] Did your changes require updates to the documentation? Have you updated
      those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

### CODE REVIEW

File: docs/source/bentocloud/reference/deployment-details.rst [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+   * - CPU Usage
+     - Displays the CPU utilization of the Deployment, which includes the total usage as well as the usage by individual components.
+   * - Memory Usage
+     - Displays the amount of memory being used over time, which includes the total usage as well as the usage by individual components.
+   * - GPU Usage
+     - Displays the GPU utilization of the Deployment, which includes the total usage as well as the usage by individual components.
+   * - GPU Memory Usage
+     - Displays the memory usage on the GPU, offering insights into how memory-intensive the GPU tasks are. It includes the total usage as well as the usage by individual components.
+   * - GPU Memory Bandwidth Usage
+     - Displays the bandwidth usage of the GPU memory, providing data on how intensively the memory is being accessed and at what rate data is being transferred. It includes the total usage as well as the usage by individual components.
```
Comment (Sherlock113) [inline docs/source/bentocloud/reference/deployment-details.rst:161]: @Haivilo Please help take a look at the Monitoring properties. Thanks!

File: docs/source/bentocloud/reference/deployment-details.rst [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+   * - Deduplication
+     - Filters out duplicate log entries. Options including "None", "Exact", "Numbers" and "Signature" provide different methods to identify and remove repeated logs.
+   * - Display results
+     - Sort log entries chronologically, either from the newest or the oldest.
+   * - Download
+     - Download logs in text or JSON format.
+
+.. note::
+
+   The **Logging** tab integrates Grafana Loki. For more information, see the `Loki documentation <https://grafana.com/docs/loki/latest/>`_.
```
Comment (Sherlock113) [inline docs/source/bentocloud/reference/deployment-details.rst:133]: @frostming Please help take a look at the these Logging properties. Thanks!

File: docs/source/bentocloud/reference/deployment-details.rst (hunk 1 above)
Comment (Haivilo) [reply docs/source/bentocloud/reference/deployment-details.rst:161]: lgtm!

**Comment by Sherlock113** [comment]
Merging this... @frostming @yetone Let me know if you have other questions and I will update it later. Thanks.

**Pull request merged**